instrumentation/httpurlconnection/library/src/main/java/io/opentelemetry/instrumentation/library/httpurlconnection/HttpUrlReplacements.java

```diff
@@ -0,0 +1,402 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.library.httpurlconnection;
+
+import static io.opentelemetry.instrumentation.library.httpurlconnection.internal.HttpUrlConnectionSingletons.instrumenter;
+
+import android.os.SystemClock;
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.library.httpurlconnection.internal.RequestPropertySetter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URLConnection;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class HttpUrlReplacements {
+
+    private static final Map<URLConnection, HttpURLConnectionInfo> activeURLConnections;
+    private static final Logger logger;
+    public static final int UNKNOWN_RESPONSE_CODE = -1;
+
+    static {
+        activeURLConnections = new ConcurrentHashMap<>();
+        logger = Logger.getLogger("HttpUrlReplacements");
+    }
+
+    public static void replacementForDisconnect(HttpURLConnection c) {
```

Comment:
We generally don't favor single-letter parameter/variable names. Let's call them `connection` or at worst `conn`. :)

Reply:
Applies to all the other methods as well.